js/webtorrent/components/mediaViewer.js

```diff
@@ -0,0 +1,23 @@
+const React = require('react')
+
+class MediaViewer extends React.Component {
+  render () {
+    const torrent = this.props.torrent
+    const ix = this.props.ix
+
+    let content
+    if (torrent.serverURL != null) {
+      content = <iframe src={torrent.serverURL + '/' + ix} sandbox='allow-same-origin' />
```

Comment:
We need the iframe to actually have an origin of `http://localhost:xxxx` so that it can make http requests to the server at that same location. Otherwise, the existence of `sandbox` makes the iframe have it's own, unique origin.